index.js

```diff
@@ -17,17 +17,102 @@ function simplify (data) {
   return transform(data.value, data.type)
 }
 
-module.exports = {
-  parse: (text) => {
-    try {
-      const parserNE = new nearley.Parser(nearley.Grammar.fromCompiled(grammar))
-      parserNE.feed(text)
-      return parserNE.results[0]
-    } catch (e) {
-      e.message = `Error parsing text '${text}'`
-      throw e
+function stringify (val) {
```

Comment:
what about adding an optional `space` parameter like in the `JSON.stringify`, to allow to "beautify" mojangson. Could be done in a separate PR